command/build.py

```diff
@@ -1,8 +1,37 @@
+import yaml as pyYaml
```

Comment:
in previous PR you used just `yaml` this is quite inconsistent naming

Reply:
@MartinBasti addressed in latest commit.